npm/translatePolicy_test.go

```diff
@@ -2,366 +2,17 @@ package npm
 
```

Comment:
It's ok to keep all the tests in translatePolicy_test.go

Within the tests we can divide them via Golang Sub-tests